### PR TITLE
Remove slackbot promotional verbiage and add redirects

### PR DESCRIPTION
**Pull request by susodapop**

### CODE REVIEW

File: src/pages/kb/user-guide/integrations-and-api/slack-bot-posting-visualizations-to-slack.md [hunk 1]
```diff
@@ -6,6 +6,9 @@ title: Slack Bot (posting visualizations to Slack)
 slug: slack-bot-posting-visualizations-to-slack
 ---
 
+{% callout danger %}
+The Redash Slackbot is now deprecated.
+{% endcallout %}
```
Comment (arikfr) [inline src/pages/kb/user-guide/integrations-and-api/slack-bot-posting-visualizations-to-slack.md:11]: Should we also remove the rest of the content on this page? And maybe link to the future forum post?

File: src/templates/DataSourcePage.jsx [hunk 2]
```diff
@@ -129,7 +129,7 @@ export default function DataSourcePageTemplate({
               <ul className="highlight-ul m-t-50">
                 <li>
                   <b>Share insights from {frontmatter.name}</b> via link or
-                  Slack or embed them wherever you need to make your
+                 or embed them wherever you need to make your
```
Comment (arikfr) [inline src/templates/DataSourcePage.jsx:132]: I think this results in two `or`s.

File: src/pages/kb/user-guide/integrations-and-api/slack-bot-posting-visualizations-to-slack.md (hunk 1 above)
Comment (susodapop) [reply src/pages/kb/user-guide/integrations-and-api/slack-bot-posting-visualizations-to-slack.md:11]: Yes linking to the forum post is the right move. I'm drafting that in gdocs now. Happy to remove the rest of the page contents as well.

File: src/templates/DataSourcePage.jsx (hunk 2 above)
Comment (susodapop) [reply src/templates/DataSourcePage.jsx:132]: Fixed 4667265891383a793f77b2295dd9eda02e99b4cb

File: src/pages/kb/user-guide/integrations-and-api/slack-bot-posting-visualizations-to-slack.md (hunk 1 above)
Comment (susodapop) [reply src/pages/kb/user-guide/integrations-and-api/slack-bot-posting-visualizations-to-slack.md:11]: Removed in 9a20101e14e3ad4e6f9c2158097f808310b0b026

**Pull request merged**